lib/plugins/aws/deployFunction.js

```diff
@@ -63,6 +65,35 @@ class AwsDeployFunction {
     }
   }
 
+  // TODO: TESTS
+  checkIfFunctionChangesBetweenImageAndHandler() {
+    const functionObject = this.serverless.service.getFunction(this.options.function);
+    const remoteFunctionPackageType = this.serverless.service.provider.remoteFunctionData
+      .Configuration.PackageType;
+
+    if (functionObject.handler && remoteFunctionPackageType === 'Image') {
+      const errorMessage = [
+        `The function "${this.options.function}" you want to update with handler was previously packaged as an image.`,
+        ' Please run "serverless deploy" to ensure consistent deploy.',
+      ];
+      throw new this.serverless.classes.Error(
+        errorMessage,
```

Comment:
If I read correctly it's an _array_, while what we should pass to `ServerlessError` class should be string

(ideally would be simply to provide this error message inline, without arrayifciation and joining - we can simply split string into parts)